pkg/semodule/policycoreutils/policycoreutils.go

```diff
@@ -0,0 +1,73 @@
+//go:build policycoreutils
+// +build policycoreutils
+
+package policycoreutils
+
+import (
+	"os/exec"
+	"strings"
+
+	"github.com/containers/selinuxd/pkg/semodule/seiface"
+	"github.com/go-logr/logr"
+)
+
+type SEModulePcuHandler struct {
+	logger logr.Logger
+}
+
+// Ensure that the test handler implements the Handler interface
+var _ seiface.Handler = &SEModulePcuHandler{}
+
+func runSeModule(opFlag, policyArg string) (string, error) {
+	cmd := exec.Command("/usr/sbin/semodule", opFlag, policyArg)
+	out, err := cmd.CombinedOutput()
+	return string(out), err
+}
+
+func NewSEModulePcuHandler(logger logr.Logger) (*SEModulePcuHandler, error) {
+	return &SEModulePcuHandler{logger: logger}, nil
+}
+
+func (smt *SEModulePcuHandler) SetAutoCommit(_ bool) {
+	// left to policycoreutils
+}
+
+func (smt *SEModulePcuHandler) Install(modulePath string) error {
+	out, err := runSeModule("-i", modulePath)
+	if err != nil {
+		smt.logger.Error(err, "semodule -i", "modulePath", modulePath)
```

Comment:
doesn't the command that was run already appear in the `err` that was returned? If so, might want to change the error message to something more human readable that we can `grep` for.